src/dotnet/ToolPackage/RestoreCommandIdentifier.cs

```diff
@@ -11,9 +11,9 @@ namespace Microsoft.DotNet.ToolPackage
     /// <summary>
     ///     Given the following parameter, a list of RestoredCommand of a NuGet package can be uniquely identified
     /// </summary>
-    internal class CommandSettingsListId : IEquatable<CommandSettingsListId>
+    internal class RestoreCommandIdentifier : IEquatable<RestoreCommandIdentifier>
```

Comment:
This should be `RestoredCommandIdentifier` (it's messing a `d`).

Reply:
When you used a spell check but still get typo out...

Reply:
:P fixing

Reply:
fixed, added the missing `d`